cards/spells/chill-touch.tex

```diff
@@ -6,30 +6,19 @@
 
 \CardTraits{Necromancy, Negative}
 
-% TODO: fix formatting to make card fit on one page and remove \footnotesize
-{\footnotesize
 \Flavor{Siphoning negative energy into yourself, your hand radiates a pale darkness.}
 
 Touch 1 living or undead creature.
-}
 
 \Action[Somatic, Verbal]{CRB}{323}{2}{Weaken Living}
 
-\DamageRoll{Negative}{\level{}d4}{spell mod}
+\DamageRoll{Negative}{\level{}d4}{spell mod} (Fortitude Save)
 
-{\footnotesize
-The target gets a basic Fortitude save.
-
-\pfsymbol{critical-failure} The target is also enfeebled 1 for 1 round.
-}
+\pfsymbol{critical-failure} Also enfeebled 1 for 1 round.
 
 \Action[Somatic, Verbal]{CRB}{323}{2}{Disorient Undead}
 
-{\footnotesize
-The target gets a basic Fortitude save.
-
-\pfsymbol{failure} The target is flat-footed for 1 round.
+Flat-footed for 1 round (Fortitude save).
```

Comment:
Why did you remove the `\pfsymbol{failure}`? How are we noting things like this down on other cards?

I tried to go for a structure like this (and then struggled with the nested saves)
```
attempt <save>
[<effect> on critical success]
[<effect> on success]
[<effect> on failure]
[<effect> on critical failure]
```

You are proposing something like
```
<effect> happens (unless <save>)

[<effect> on critical success]
[<effect> on success]
[<effect> on failure]
[<effect> on critical failure]
```

I wonder whether your approach works for _basic_ saving throws better :thinking: 

Overall, I think I would prefer something like the following (although that does not look very nice, I'm afraid):

```latex
(Fortitude Save)

\pfsymbol{failure} Flat-footed for 1 round.

\pfsymbol{critical-failure} Also fleeing for 1 round (Will save).
```

Reply:
But if the effect is dealing damage, the typical structure is:

```
<damage> happens (modified by <save>)
```

I think a good argument in favor of this structure is that for effects on multiple targets, on rolls the damage once, and then every target rolls their save. So this format shows from left to right what to roll in which order.

So from my perspective, we can generalize from <damage> to all effects and have the following format:

```
<effects> (modified by <save>)

<alternative or additional effect for certain outcomes>
```

And then we can leave out one of `<effects> or <alternative or additional effect for certain outcomes>` as needed.

From a drama perspective, I think the effect - save - actual effect sequence is also nice, because the person rolling the save knows what's at stake. There's a difference between "roll a save" and "roll a save to avoid dying" at the table ...

Reply:
Do you want to keep the effect box at all in this case?

Reply:
I don't know :(

for this card, on the one hand, since the action for living targets has an effect box, it would be nice to also have one in the action for undead targets.

on the other hand, since "fleeing for 1 round" has not effect box, it is nice that "enfeebled 1 for one round" doesn't have one either.

(what is the difference between "enfeebled 1" and "enfeebled 1 for 1 round" anyway?)